### DISCUSSION TITLE
Same config, different resoults

### DISCUSSION THREAD

**Discussion by thiagottjv** [madMAx43v3r/chia-plotter#520] [Q&A]
Hi guys, me and a friend started plotting together, we bought the same nvmes, same HDs, same CPU.
The problem is that he is plotting super slower then me. My phase 1 is finishing in 585s, his is finishing at 950s (the other phases keep the pattern).

Our configs:
Ryzen 9 5900x, 2x NVME 2Tb 970 EVO Plus in RAID0 (tmp dir), same end HD and same OS (ClearLinux Server with exactly same install).

The differences:
MoBo: 
Mine: Gigabyte Aorus x570 wifi ITX | His: Asus x570 tuf gaming Plus
RAM:
Mine: XPG Spectrix D41, 32GB (2x16GB), 3600MHz (running at 3066MHz) | His:  PATRIOT VIPER STEEL 32GB 3200MHZ (running at 3066MHz)

Both CPU speeds are averaging at 4.5GHz.
Both RAM are at 3066MHz (confirmed on the OS).
Both NVMEs are at x4 (confirmed on the OS).

Temps are within reason... NVMEs at 65 to 70 celcius, CPU at 60C.

We don`t know what else to do... the times are very different. 

Thanks for any input.

**Comment by vladaepro** [comment]
Try to cool down nvmes, i have one throttling badly at 70c. Also, watch at cpu usage and iowait during plotting - i'm using htop and iostat -m 5, and you'll see which of two is bottleneck
